pkg/api/handlers/create_cluster.go

```diff
@@ -49,7 +49,8 @@ func (d *createCluster) Handle(params operations.CreateClusterParams, principal
 	k8sutil.EnsureNamespace(d.Kubernetes, d.Namespace)
 	kluster, err = d.Kubernikus.Kubernikus().Klusters(d.Namespace).Create(kluster)
 	if err != nil {
-		glog.Errorf("Failed to create cluster: %s", err)
+		logger.Log("msg", "failed to create cluster", "err", err)
```

Comment:
Not sure if this already part of the context but maybe we should add the cluster name here?

Reply:
Added `kluster` and `project` values.